pyrsa/vis/mds_plot.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
```

Comment:
We don't have automatic api documentation anymore, so a new module (or renamed) requires adapting the narrative documentation (rst) as well

Reply:
Just included them. BTW, why can't I see the pylint etc. checks on my end any more? Is it just me, or everyone?

Reply:
Not just you, it's been turned off (maybe something to discuss in the workshop?)